release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -4506,4 +4506,25 @@ $ oc adm release info 4.13.56 --pullspecs
 
 [id="ocp-4-13-56-updating_{context}"]
 ==== Updating
+To update an existing {product-title} 4.13 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+// 4.13.57
+[id="ocp-4-13-57_{context}"]
+=== RHSA-2025:3780 - {product-title} 4.13.57 bug fix and security updates
+
+Issued: 17 April 2025
+
+{product-title} release 4.13.57, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:3780[RHSA-2025:3780] advisory. 
```

Comment:
```suggestion
{product-title} release 4.13.57, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:3780[RHSA-2025:3780] advisory. There are no RPM packages for this update.
```

Aligns with past ways of handling a dropped release: https://docs.redhat.com/en/documentation/openshift_container_platform/4.16/html-single/release_notes/index#ocp-4-16-13_release-notes